src/rpc/misc.cpp

```diff
@@ -234,6 +235,9 @@ UniValue validateaddress(const JSONRPCRequest& request)
     return ret;
 }
 
+// Needed even with !ENABLE_WALLET, to pass (ignored) pointers around
+class CWallet;
```

Comment:
This doesn't actually seem to be needed since CWallet is also forward declared in init.h, but maybe it is better to keep it.